web_src/css/install.css

```diff
@@ -18,17 +20,12 @@
   width: auto;
 }
 
-.page-content.install form.ui.form .title {
-  margin-left: 30%;
-  padding-left: 5px;
-}
-
 .page-content.install form.ui.form input {
   width: 60%;
 }
 
 .page-content.install form.ui.form details.optional.field[open] {
-  border-bottom: 1px solid var(--color-secondary);
+  border-bottom: 1px dashed var(--color-secondary);
```

Comment:
We rarely use dashed borders, is it justified here?

Reply:
Yes, to distinguish it from the parent header's solid border. You can take a try on that page. 

It's not a perfect solution but it is simple and quick, and makes it a little better. If you really don't like, it can be reverted.

Reply:
If you have a screenshot, please show. It's no big issue for me.

Reply:
Seems alright